_rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+max_width=80
```

Comment:
must this be in the root?

Reply:
also please remove `flatc_debian_stretch`

Reply:
oh whoops, I was saving this for a future PR, I'll delete them both